src/overlays/misc/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -156,31 +156,39 @@ void KaleidoScope_DrawItemSelect(PlayState* play) {
                 PRINTF("now=%d  ccc=%d\n", cursorPoint, cursorItem);
 
                 do {
+                    // if input says move left
                     if (pauseCtx->stickAdjX < -30) {
+                        // if not left-most
                         if (pauseCtx->cursorX[PAUSE_ITEM] != 0) {
```

Comment:
comment 1) I cant explain why but "says move left" sounds weird to me. Id prefer something like "if stick is on left side", or simply just no comment at all 

comment 2) left most what, slot I guess?